docs/kqueen.rst

```diff
@@ -422,12 +425,27 @@ For more information, see `Issues <https://github.com/Azure/AKS/issues/3>`_, and
 To manually add an existing Kubernetes cluster:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Log in to the KQueen web UI.
-2. Create ``Manual Provisioner``.
-3. In the ``Create Cluster`` tab choose predefined manual provisioner and attach a valid Kubernetes configuration file.
+#. Log in to the KQueen web UI.
+#. Create ``Manual Provisioner``.
```

Comment:
#. Click ``Create Provisioner`.
#. Enter the provisioner name, select ``Manual Engine`` from the ``Engine`` drop-down list, and then click   ``Submit``.
#. Click ``Deploy Cluster``.
#. On the ``Deploy Cluster`` page, define the cluster name, select a predefined manual provisioner, and attach a valid Kubernetes configuration file. Then click ``Submit``.

Reply:
done